generators/client/templates/react/src/main/webapp/app/shared/reducers/profile.ts.ejs

```diff
@@ -0,0 +1,53 @@
+<%#
+ Copyright 2013-2018 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import axios from 'axios';
+
+import { SUCCESS } from 'app/shared/reducers/action-type.util';
+
+export const ACTION_TYPES = {
+  GET_PROFILE: 'profile/GET_PROFILE'
+};
+
+const initialState = {
+  activeProfiles: [],
+  ribbonEnv: '',
+  isInProduction: true,
+  isSwaggerEnabled: false
+};
+
+export default (state = initialState, action) => {
```

Comment:
I think this was already in authentication or admin reducer can you remove it there

Reply:
I can't find it in either of them, should I move it in authentication reducer then ?